keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/api/ec2/Ec2JsonTestConfiguration.kt

```diff
@@ -0,0 +1,45 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.api.support.ExtensionRegistry
+import com.netflix.spinnaker.keel.api.support.register
+import com.netflix.spinnaker.keel.ec2.jackson.registerEc2Subtypes
+import com.netflix.spinnaker.keel.ec2.jackson.registerKeelEc2ApiModule
+import com.netflix.spinnaker.keel.jackson.registerKeelApiModule
+import org.springframework.boot.jackson.JsonComponentModule
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+
+/**
+ * Configures the primary object mapper with auto-wired custom JSON serializers and deserializers
+ * via [JsonComponentModule].
+ */
+@Configuration
+@ComponentScan(basePackages = ["com.netflix.spinnaker.keel.ec2.jackson"])
+class Ec2JsonTestConfiguration {
```

Comment:
@robfletcher I've attempted to create a new module `keel-ec2-test-support` with this class, and add a dependency in the other modules, so it could be reused, however I ran into Spring app initialization errors because the new module depended on `keel-ec2-plugin` which brought in a bunch of undesired Spring beans. If you have any ideas on how to approach this to avoid the redundancy, I'd love your thoughts.

Reply:
I don't really get what this class is for. Wouldn't any Spring tests that needed the EC2 plugin stuff to be in place be getting all of this from main configuration? Why do you need a special test configuration class?

Reply:
As discussed in chat, I had incorrectly assumed, based on other tests annotated with `@SpringBootTest`, that you had to manually specify `KeelApplication::class` as one of the configuration classes for it to load, and my rationale was to create a configuration that was much smaller, containing essentially the `ObjectMapper` that these kinds of tests need, especially since they were not loading a Spring application context before (I was wary of extra time to run the tests).

I'll try to use the main configuration and create an `application.properties` config file for each module where we have these such that I can at least disable certain things, especially liquibase migrations.

Reply:
PR updated to use `KeelApplication` configuration for all new `SpringBootTest` tests.